tasks/terraform_apply.rb

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative '../../ruby_task_helper/files/task_helper.rb'
+require_relative '../lib/cli_helper.rb'
+require 'json'
+require 'open3'
+
+class TerraformApply < TaskHelper
+  def apply(opts)
+    dir = File.expand_path(opts[:dir], opts[:_boltdir]) if opts[:dir]
```

Comment:
The ruby-var `dir` needs to be set in order to expand other paths in this file. Also, I think `opts[:_boltdir]` is only available for tasks that are used as plugins (https://github.com/puppetlabs/bolt/blob/master/lib/bolt/plugin/module.rb#L173), which makes sense since this could be executing remotely. I think we should just require the `dir` key, and require it to be an absolute path.

Reply:
Good point. I forgot about that. Updated.